test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios');
+const chai = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const { expect } = chai;
+chai.use(require('chai-subset'));
+
+const urlBase = 'https://api.github.com/users';
+const keyPath = axios.create({
+  path: urlBase,
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+
+const params = {
+  per_page: 40
+};
+
+describe.only('Stage 12 - Query parameters: Users', () => {
+  let users;
+  before(async () => {
+    users = await keyPath.get(`${urlBase}`, params);
```

Comment:
Try with:
```suggestion
    users = await keyPath.get(`${urlBase}`, {params});
```

Reply:
The force is with you! Thanks a lot, you saved me!